webapp/src/components/call_widget/component.tsx

```diff
@@ -588,6 +588,13 @@ export default class CallWidget extends React.PureComponent<Props, State> {
         const currentID = this.props.currentUserID;
         const isSharing = sharingID === currentID;
 
+        let fill = '';
+        if (isSharing) {
+            fill = 'rgba(210, 75, 78, 1)';
```

Comment:
Probably we should change this to `var(--center-channel-color)` while we're at it?  (or whatever var this rgba is)

Reply:
Looks like that would be the `--dnd-indicator`. Let me update :+1: